opentile/ndpi_tiler.py

```diff
@@ -615,10 +615,16 @@ def _crop_to_tiles(
         Dict[Point, bytes]:
             Created tiles ordered by tile coordinate.
         """
-        tiles: List[bytes] = self._jpeg.crop_multiple(
-            frame,
-            frame_job.crop_parameters
-        )
+        try:
+            tiles: List[bytes] = self._jpeg.crop_multiple(
+                frame,
+                frame_job.crop_parameters
+            )
+        except JpegCropError:
```

Comment:
This except clause seems pointless to me.

Reply:
The point is to give a more detailed exception including what frame that failed, without passing additional data to jpeg crop_multiple().